spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configuration-watcher/src/main/java/org/springframework/cloud/kubernetes/configuration/watcher/SecretsWatcherChangeDetector.java

```diff
@@ -58,16 +58,6 @@ public SecretsWatcherChangeDetector(CoreV1Api coreV1Api, ConfigurableEnvironment
 		this.k8SConfigurationProperties = k8SConfigurationProperties;
 	}
 
-	protected boolean isSpringCloudKubernetesSecret(V1Secret secret) {
```

Comment:
I did not remove anything here, slightly adjusted so that code looks the same visually in configmap and secrets implementation